vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -321,8 +321,22 @@ function Update-WindowsFeatures {
 }
 
 function Update-Registry {
+
+    param (
+        $containerRuntime
+    )
+
+    # if multple LB policies are included for same endpoint then HNS hangs.
+    # this fix forces an error
     Write-Host "Enable a HNS fix in 2021-2C"
     reg add HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\hns\State /v HNSControlFlag /t REG_DWORD /d 1
+
+    # Enables DNS resolution of SMB shares for containerD
+    # https://github.com/kubernetes-sigs/windows-gmsa/issues/30#issuecomment-802240945
+    if ($containerRuntime -eq 'containerd') {
+        Write-Host "Apply SMB Resolution Fix for containerD"
```

Comment:
I have a PR to change Write-Host to Write-Log to keep the same format. Since you are adding a new one, could you help to change this to Write-Log at least this line in your PR?